### PR TITLE
Bug/81730 IPO information email in functional role not working

### PR DESCRIPTION
**Pull request by VRiveland**
Added handling of multiple emails in email and information email fields in functional roles and added information emails to the list of people added to outlook meeting.

Completes: [AB#81730](https://statoildeveloper.visualstudio.com/b6d97460-2e53-47ae-aad4-b41396d90828/_workitems/edit/81730)

### CODE REVIEW

**Review by llykkelarsen — APPROVED** [review]
👍

**Comment by paal123** [comment]
@janid1967 These classes are terrible and should be refactored. I know this PR is only a minor adjustment, but please not @VRiveland that these classes breaks most of the rules we have in backend development

File: src/Equinor.ProCoSys.IPO.Command/InvitationCommands/CreateInvitation/CreateInvitationCommandHandler.cs [hunk 1]
```diff
@@ -163,8 +163,21 @@ public async Task<Result<int>> Handle(CreateInvitationCommand request, Cancellat
                         participant.SortKey));
                     if (fr.UsePersonalEmail != null && fr.UsePersonalEmail == false && fr.Email != null)
                     {
-                        participants.Add(new BuilderParticipant(ParticipantType.Required,
-                            new ParticipantIdentifier(fr.Email)));
+                        var emails = fr.Email.Split(";");
```
Comment (paal123) [inline src/Equinor.ProCoSys.IPO.Command/InvitationCommands/CreateInvitation/CreateInvitationCommandHandler.cs:166]: Create helper methods here instead. With only one line in here
participants.AddRange(SplitAndCreateBuilderParticipantsFromEmailList(fr.Email));

and same for InformationEmail
participants.AddRange(SplitAndCreateBuilderParticipantsFromEmailList(fr.InformationEmail));

File: src/Equinor.ProCoSys.IPO.Command/InvitationCommands/EditInvitation/EditInvitationCommandHandler.cs [hunk 2]
```diff
@@ -269,8 +269,22 @@ private async Task<List<CommPkg>> GetCommPkgScopeAsync(IList<string> commPkgNos,
                     
                     if (fr.UsePersonalEmail != null && fr.UsePersonalEmail == false && fr.Email != null)
                     {
-                        participants.Add(new BuilderParticipant(ParticipantType.Required,
-                            new ParticipantIdentifier(fr.Email)));
+
```
Comment (paal123) [inline src/Equinor.ProCoSys.IPO.Command/InvitationCommands/EditInvitation/EditInvitationCommandHandler.cs:272]: I would like to see update tests for your changes

**Review by paal123 — APPROVED** [review]
With suggestions, see comments

**Review by paal123 — CHANGES_REQUESTED** [review]
See comments

File: src/Equinor.ProCoSys.IPO.Command/InvitationCommands/EditInvitation/EditInvitationCommandHandler.cs (hunk 2 above)
Comment (janid1967) [reply src/Equinor.ProCoSys.IPO.Command/InvitationCommands/EditInvitation/EditInvitationCommandHandler.cs:272]: I guess you mean Unit tests @paal123 .. and I agree

**Review by janid1967 — CHANGES_REQUESTED** [review]
Build breaks. Test issue?

**Comment by VRiveland** [comment]
> Build breaks. Test issue?

Nah, kubernetes error, same as the comm app earlier in the week. Should be fixed soon.

**Review by janid1967 — APPROVED** [review]

**Pull request merged**